pkg/hsm/awscloudhsm/cloudhsm.go

```diff
@@ -161,6 +162,7 @@ func (c *Client) GenerateAndSignMetadata(request hsm.GenerateMetadataRequest) (s
 		"--hsm-saml-signing-key-label", request.SamlSigningKeyLabel,
 		samlSigningOption, tmpSAMLSigningCertPath,
 		"--supplied-saml-encryption-cert-file", tmpSAMLEncryptionCertPath,
+		"--validityDays", request.Data.ValidityDays,
```

Comment:
The MetadataGenerator `--validityDays` option is an Integer, I'm guessing picocli will marshall this golang string into an Integer, see https://github.com/alphagov/verify-metadata-controller/blob/a80b88661f2b1d4279bdc8b91d32638bf58f2e96/mdgen/src/main/java/uk/gov/ida/mdgen/MetadataGenerator.java#L122-L123